### ISSUE TITLE
Transform All Columns

### ISSUE THREAD

**Issue by allthebits** [OpenRefine/OpenRefine#1724]
When Transforming ALL rows of data:

![image](https://user-images.githubusercontent.com/2952327/45162956-2e636000-b1bd-11e8-9ed5-a0f4685fd872.png)

add a checkbox in the popup that allows you to "select all columns" to apply the transformation to all rows of data and all columns of data regardless of the column name, preventing the column names from being statically defined within the JSON representation of the transformation.

This request is based on the following discussion thread from the [OpenRefine google group](https://groups.google.com/forum/#!topic/openrefine/C3W-besDx4U)

**Comment by thadguidry** [comment]
Done !  This was released as part of OpenRefine 2.7 !  Closing this issue now.

https://github.com/OpenRefine/OpenRefine/commit/20acc17c05c3d76a643908fe8918d788abc2cf6f#diff-350f28f0f68f7da753d1e8b086362e94

**Comment by ostephens** [comment]
I think it would be an additional option on this screen:

<img width="822" alt="screenshot 2018-11-21 at 15 31 12" src="https://user-images.githubusercontent.com/576174/48851277-8401de80-eda2-11e8-8f63-11f618222267.png">

**Comment by thadguidry** [comment]
I still have a need for this... a few datasets I imported recently, I needed to apply a Transform to all the columns except column 1 & 2.  Painful without the Column selector idea above from @ostephens to interactively ask which columns to apply the Transform expression on.

@wetneb I think this is probably a good candidate feature to prioritize, it comes up too often and would eliminate a lot of tedious user clicking currently that's really unnecessary if we added the column selector to All -> Transform.
